parkings/importers/parking_areas.py

```diff
@@ -0,0 +1,201 @@
+            'gml': 'http://www.opengis.net/gml/3.2',
+        }
+        self.overwrite = overwrite
+        self.verbosity = verbosity
+
+        self.refusals = 0
+        self.overwrites = 0
+        self.created = 0
+
+    def import_areas(self):
+        start = time.time()
+
+        self._create_areas(self._parse_areas())
+
+        if self.verbosity >= 1:
+            if self.refusals != 0:
+                print(
+                    'Refused to overwrite data for %s areas because'
+                    ' --overwrite is not set.' % self.refusals
+                )
+            if self.overwrites != 0:
+                print('Overwrote data for %s parking areas.' % self.overwrites)
+            if self.created != 0:
+                print('Created %s new parking areas.' % self.created)
+            if (self.created + self.refusals + self.overwrites) == 0:
+                print('Nothing to do.')
+
+            print('Finished in %s seconds.' % round(time.time() - start, 2))
+
+    def _create_areas(self, area_dicts):
+        area_dicts_count = len(area_dicts)
+
+        for index, area_dict in enumerate(area_dicts):
+            if self.verbosity >= 1:
+                progress = 100 * ((index + 1) / area_dicts_count)
+                sys.stdout.write('\rCreating areas %d%%' % progress)
+
+            try:
+                parking_area = ParkingArea.objects.get(
+                    external_id=area_dict['external_id'],
+                )
+                self._update_parking_area(area_dict, parking_area)
+            except ParkingArea.DoesNotExist:
+                parking_area = self._create_parking_area(area_dict)
+
+            parking_area.save()
+        if self.verbosity >= 1:
+            print('')
+
+    def _parse_areas(self):
+        '''
+        :returns: A list of all the found areas as a dict.
+        '''
+        response = self._download()
+        root = etree.fromstring(bytes(response.getvalue(), 'UTF-8'))
+        members = self._separate_members(root)
+
+        parsed_members = []
+        for index, member in enumerate(members):
+            if self.verbosity >= 1:
+                progress = 100 * ((index + 1) / len(members))
+                sys.stdout.write('\rParsing %d%%' % progress)
+
+            parsed_members.append(self._parse_member(member))
+        if self.verbosity >= 1:
+            print('')
+
+        return parsed_members
+
+    def _create_parking_area(self, area_dict):
+        parking_area = ParkingArea(
+            external_id=area_dict['external_id'],
+            areas=area_dict['areas'],
+            space_amount_estimate=area_dict['space_amount_estimate'],
+        )
+        self.created += 1
+        return parking_area
+
+    def _update_parking_area(self, area_dict, parking_area):
+        space_amount_estimate = area_dict['space_amount_estimate']
+        if space_amount_estimate:
+            parking_area.space_amount_estimate = space_amount_estimate
```

Comment:
Nice catch! Actually a few lines down this is done correctly so this is just leftover code.